test/jdk/javax/swing/JSpinner/4670051/JSpinnerFieldUnderCursorTest.java

```diff
@@ -0,0 +1,281 @@
+                });
+
+                // Cursor at Day field.
+                // Increment Day
+                initValue = getSpinnerValue();
+                mousePressOnUpButton();
+                upValue = getSpinnerValue();
+                verifyDayIncrement();
+                // Decrement Day
+                initValue = getSpinnerValue();
+                mousePressOnDownButton();
+                downValue = getSpinnerValue();
+                verifyDayDecrement();
+
+                // Cursor at Month Field
+                pressRightArrowKey();
+                // Increment Month
+                initValue = getSpinnerValue();
+                mousePressOnUpButton();
+                upValue = getSpinnerValue();
+                verifyMonthIncrement();
+                // Decrement Month
+                initValue = getSpinnerValue();
+                mousePressOnDownButton();
+                downValue = getSpinnerValue();
+                verifyMonthDecrement();
+
+                // Cursor at Year Field
+                pressRightArrowKey();
+                // Increment Year
+                initValue = getSpinnerValue();
+                mousePressOnUpButton();
+                upValue = getSpinnerValue();
+                verifyYearIncrement();
+                // Decrement Year
+                initValue = getSpinnerValue();
+                mousePressOnDownButton();
+                downValue = getSpinnerValue();
+                verifyYearDecrement();
+
+                if (passed) {
+                    System.out.println("Test Passed");
+                } else {
+                    throw new RuntimeException("Test Failed as one or more cases failed");
+                }
+            } finally {
+                SwingUtilities.invokeAndWait(JSpinnerFieldUnderCursorTest::disposeFrame);
+            }
+        }
+    }
+
+    private static Date getSpinnerValue() throws Exception {
+        SwingUtilities.invokeAndWait(() -> spinnerValue = (Date) spinner.getValue());
+        return spinnerValue;
+    }
+
+    public static void pressRightArrowKey() {
+        robot.keyPress(KeyEvent.VK_RIGHT);
+        robot.keyRelease(KeyEvent.VK_RIGHT);
+    }
+
+    public static void mousePressOnUpButton() {
+        robot.mouseMove(spinnerUpButtonCenter.x, spinnerUpButtonCenter.y);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+    }
+
+    public static void mousePressOnDownButton() {
+        robot.mouseMove(spinnerDownButtonCenter.x, spinnerDownButtonCenter.y);
+        robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+        robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+    }
+
+
+    public static boolean compareDates() {
+        return (expected.get(Calendar.DATE) == actual.get(Calendar.DATE)) &&
+                (expected.get(Calendar.MONTH) == actual.get(Calendar.MONTH)) &&
+                (expected.get(Calendar.YEAR) == actual.get(Calendar.YEAR));
```

Comment:
```suggestion
        return (expected.get(Calendar.DATE) == actual.get(Calendar.DATE))
               && (expected.get(Calendar.MONTH) == actual.get(Calendar.MONTH))
               && (expected.get(Calendar.YEAR) == actual.get(Calendar.YEAR));
```
Otherwise, this looks inconsistently to other places where you put the operator on the wrapped line.